<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Combo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>简单ComboBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnInitLeft">
        <property name="text">
         <string>初始化列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnClearLeft">
        <property name="text">
         <string>清空列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkEditable">
        <property name="text">
         <string>可编辑</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QComboBox" name="comboLeft">
        <item>
         <property name="text">
          <string>南京</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../《Qt 6 C++开发指南》全书示例源码、演示软件及使用说明-20231218更新/03《Qt 6 C++开发指南》全书示例源码--qmake练习版 V1.1/Chap04_Widgets/samp4_09Combobox/icons/UNIT.ICO</normaloff>../../《Qt 6 C++开发指南》全书示例源码、演示软件及使用说明-20231218更新/03《Qt 6 C++开发指南》全书示例源码--qmake练习版 V1.1/Chap04_Widgets/samp4_09Combobox/icons/UNIT.ICO</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>杭州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>苏州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小邓</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>有用户数据的ComboBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnInitRight">
        <property name="text">
         <string>初始化城市+区号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboRight">
        <property name="placeholderText">
         <string>请选择城市</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnClearUnder">
        <property name="text">
         <string>清空文本</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
